src/__tests__/controllers/project.test.js

```diff
@@ -1 +1,65 @@
-test.only('some test', () => {});
+const request = require('supertest');
+const Database = require('../../db');
+const app = require('../../app');
+const { Volunteer, Organization } = require('../../models/user');
+const Project = require('../../models/project');
+const exampleData = require('./exampleData.json');
+
+const db = new Database(process.env.DB_TEST_URL);
+
+describe('connecting,clearing and preloading the database', () => {
+  beforeAll(async () => {
+    await db.getConnection();
+    await db.dropDatabase();
+  });
+
+  beforeEach(async () => {
+    await db.dropDatabase();
+  });
+
+  afterAll(async () => {
+    await db.dropDatabase();
+    db.closeConnection();
+  });
+
+  describe('GET /api/projects/filter', () => {
+    test('Should filter projects by location or creator', async () => {
+      const organization = await Organization.create(exampleData.organization);
+      const project = await Project.create({
+        creator: organization._id,
+        ...exampleData.project,
+      });
+
+      const response = await request(app)
+        .get('/api/projects/filter')
+        .set('Content-Type', 'application/json')
+        .query({ address: `${project.address}` });
+
+      expect(response.header['content-type']).toContain('application/json');
+      expect(response.statusCode).toBe(200);
+      expect(response.body[0].title).toBe(project.title);
+    });
+  });
+
+  describe('GET /api/projects/app/:id', () => {
```

Comment:
Check the description and naming of the route. I think it's a POST request and we changed the naming to `applications`.